testing/tx.go

```diff
@@ -52,7 +52,7 @@ func WaitForTx(ctx context.Context, be bind.DeployBackend, tx *types.Transaction
 		simulated.Commit()
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, time.Minute)
```

Comment:
It makes sense, given that the sepolia block time is longer than the local devnet. In the future it might be useful to add a flag to toggle between different setting